test/flow-types/regression-1114.js.flow

```diff
@@ -25,11 +25,6 @@ test.cb(t => {
 	t.end();
 });
 
-test.beforeEach(t => {
-	// $ExpectError: Unknown property "context"
```

Comment:
I thought about keeping the beforeEach, but then I thought it has nothing to with the issue #1114. But maybe it has, I didn't dig deeper.